pytorch_lightning/accelerators/ddp2_accelerator.py

```diff
@@ -123,9 +123,6 @@ def ddp_train(self, process_idx, mp_queue, model):
             process_idx:
             mp_queue: multiprocessing queue
             model:
```

Comment:
@akihironitta update this everywhere?
```suggestion
            process_idx: current process rank
            mp_queue: multiprocessing queue
            model: pointer to current :class:`LightningModule`
```

Reply:
@rohitgr7 Sure. Thank you very much for your help!